zk_prover/examples/gen_solvency_verifier.rs

```diff
@@ -50,6 +50,12 @@ fn main() {
 
     let circuit = SolvencyCircuit::<N_ASSETS, N_BYTES>::init(merkle_sum_tree, asset_sums);
 
+    // generate a universal trusted setup for testing, along with the verification key (vk) and the proving key (pk).
```

Comment:
Can you mention it in the readme ? both for `gen_solvency_verifier` and `gen_inclusion_verifier`